README.md

```diff
@@ -134,6 +134,17 @@ Active Storage, with its included JavaScript library, supports uploading directl
 | `direct-upload:end` | `<input>` | `{id, file}` | A direct upload has ended. |
 | `direct-uploads:end` | `<form>` | None | All direct uploads have ended. |
 
+**NOTE:**
+If you are using Microsoft Azure Storage, you will need to set required header by hooking
+into the request before it starts:
+
+```javascript
+addEventListener("direct-upload:before-storage-request", event => {
+  const { xhr } = event.detail
+  xhr.setRequestHeader("x-ms-blob-type", "BlockBlob")
+})
+```
```

Comment:
Correct! With the headers implementation in place this is not necessary. Will change it